demo/src/screens/componentScreens/ImageScreen.tsx

```diff
@@ -1,19 +1,19 @@
 import React, {Component} from 'react';
-import {View, Text, Image, Colors} from 'react-native-ui-lib';
+import {View, Text, Image, Colors, Assets} from 'react-native-ui-lib';
 import {renderBooleanOption, renderRadioGroup, renderSliderOption} from '../ExampleScreenPresenter';
 
-import cameraIcon from '../../assets/icons/cameraSelected.png';
```

Comment:
This is causing an issue when enabling the "Show Overlay Content" toggle.

Reply:
Fixed